includes/discovery/bgp-peers.inc.php

```diff
@@ -4,231 +4,245 @@
     // Discover BGP peers
     echo 'BGP Sessions : ';
 
-    $bgpLocalAs = trim(snmp_walk($device, '.1.3.6.1.2.1.15.2', '-Oqvn', 'BGP4-MIB', $config['mibdir']));
+    if( key_exists('vrf_lite_cisco', $device) && (count($device['vrf_lite_cisco'])!=0) ){
+        $vrfs_lite_cisco = $device['vrf_lite_cisco'];
+    }
+    else{
+        $vrfs_lite_cisco = array(array('context_name'=>null));
```

Comment:
I expect this is supposed to cover non vrf_lite setups but It looks like it breaks it. You're creating an empty array and looping though it so no bgp peers will be discovered.